src/test/perl/test-caffileeditor_source.t

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use FindBin qw($Bin);
+use lib "$Bin/modules";
```

Comment:
I don't see any dummy LC modules and removing this `use lib` doesn't help if I didn't do something wrong...